hot-reload-devtools/src/main/kotlin/org/jetbrains/compose/reload/jvm/tooling/sidecar/DtSidecarWindow.kt

```diff
@@ -198,3 +226,50 @@ fun DtSidecarWindowContent(
         )
     }
 }
+
+/**
+ * Version of DtSidecarWindowContent not using size animations, for use when transparent windows are not supported
+ */
+@Composable
+fun DtSidecarWindowNonAnimatedContent(
+    isExpanded: Boolean = true,
+    isExpandedChanged: (isExpanded: Boolean) -> Unit = {},
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier.fillMaxSize(),
+        horizontalArrangement = Arrangement.End,
+    ) {
+        Box(
+            modifier = Modifier.Companion
+                .animatedReloadStatusBorder(
+                    shape = DevToolingSidecarShape,
+                    idleColor = if (isExpanded) Color.LightGray else Color.Transparent
+                )
+                .clip(DevToolingSidecarShape)
+                .background(DtColors.applicationBackground)
+                .weight(1f, fill = false),
+            contentAlignment = Alignment.TopEnd,
+        ) {
+            if (!isExpanded) {
```

Comment:
Have you thought about other approaches to disabling the animation? I see that this duplicates code a little. 
The issue is not the code duplication, but the fact that the code will be duplicated for host platforms. I might see myself easily maintaining this code, testing on macos and then forgetting to test on linux as well leading to inconsistencies which we would have to rely on manual QA to find 🤔

Reply:
I have inititally modified the `DtSidecarWindowContent` function to disable/enable animations as needed, but I saw two issues with that:
- It was making the function complex, plus having an `AnimatedContent` that sometimes doesn't animate didn't feel right, with `EnterAnimation.None.togetherWith(ExitAnimation.None)`
- I felt having a separate method makes it easier to remove later once the workaround is not longer required

But if you think it's better not to have the duplicate, I am happy to do it the other way.